pkgs/modules/nix/default.nix

```diff
@@ -0,0 +1,15 @@
+{ pkgs, ... }: {
+  id = "nix";
+  name = "Nix";
+
+  replit.languageServers.nixd = {
+    name = "nixd";
+    language = "nix";
+    start = "${pkgs.nixd}/bin/nixd";
+    extensions = [ ".nix" ];
+
+    configuration = {
+      options.enable = false;
```

Comment:
What's this part for? I would think we'd want to enable it by default since the way we enable modules right now is by installing them.

Reply:
The language server can provide completion for NixOS modules. I figured we wanted to turn that specific part off right now since we don't want/need completions for things like `config.services.*`.

It is configurable, though and we could enable completion for Replit config in the future!

Reply:
See this section for more info:

https://github.com/nix-community/nixd/blob/main/docs/user-guide.md#options